poc/nvidia/cleanup.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -e
+
+echo -e "Cleaning up MicroK8s"
+sudo snap remove microk8s --purge
```

Comment:
Maybe we can add message that dss was successfully removed.

Reply:
pushed a commit